care/emr/migrations/0061_alter_medicationrequest_dosage_instruction.py

```
@@ -0,0 +1,18 @@
+# Generated by Django 5.1.3 on 2025-01-02 20:13
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('emr', '0060_alter_medicationrequest_dosage_instruction'),
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)